pkg/plugins/common/kustomize/v2/scaffolds/internal/templates/config/rbac/crd_editor_role.go

```diff
@@ -65,7 +65,14 @@ func (f *CRDEditorRole) SetTemplateDefaults() error {
 	return nil
 }
 
-const crdRoleEditorTemplate = `# permissions for end users to edit {{ .Resource.Plural }}.
+const crdRoleEditorTemplate = `# This rule is not used by the project {{ .ProjectName }} itself.
+# It is provided to allow the cluster admin to help manage permissions for users.
+#
+# Allows users to modify all custom resources for {{ .ProjectName }} on the cluster.
+# This role enables users to create, update, and delete resources,
+# making it suitable for team members who need to manage resources
+# but should not control RBAC or manage permissions for others.
```

Comment:
```suggestion
# This role permits users to view and modify resources within the {{ .Resource.QualifiedGroup }} across the cluster.
# It grants permissions to create, update, and delete these resources, making it suitable for team members
# who need to manage resources but should not control RBAC or manage permissions for others.
```

```suggestion
# This role allows users to view and modify {{ .Resource.QualifiedGroup }} on the cluster.
# Grants permissions to create, update, and delete resources within the {{ .Resource.QualifiedGroup }} across the cluster.
```

I think we could be more clear.
We scaffold a file for each API/CRD
@damsien could we do this update in the 3 scenarios?

Reply:
Yes I think it's better to be clear about what permissions will be given in the description. Also, describe for who is this role intended for is a good idea for me!